Application/EdFi.Ods.Api.IntegrationTestHarness/Program.cs

```diff
@@ -5,268 +5,47 @@
 
 using System;
 using System.Collections.Generic;
-using System.Configuration;
 using System.IO;
-using System.Linq;
-using System.Threading;
-using Castle.MicroKernel.Registration;
-using Castle.Windsor;
-using CommandLine;
-using EdFi.Admin.DataAccess.Contexts;
-using EdFi.Admin.DataAccess.Models;
-using EdFi.Ods.Common.Configuration;
-using EdFi.Security.DataAccess.Contexts;
-using EdFi.Security.DataAccess.Repositories;
-using Microsoft.Owin.Hosting;
 using Newtonsoft.Json;
-using EdFi.Ods.Sandbox.Repositories;
-using EdFi.Ods.Sandbox.Provisioners;
-using Newtonsoft.Json.Serialization;
+using System.Threading.Tasks;
+using System.Reflection;
+using log4net.Config;
+using log4net;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Configuration;
+using Autofac.Extensions.DependencyInjection;
+using Microsoft.AspNetCore.Hosting;
 
 namespace EdFi.Ods.Api.IntegrationTestHarness
 {
     public class Program
     {
-        private static TestHarnessConfiguration _testHarnessConfiguration = new TestHarnessConfiguration();
-
-        internal static int Main(string[] args)
+        public static async Task Main(string[] args)
         {
-            var result = ReturnStatus.Success;
-
-            Options options = null;
-
-            Parser.Default.ParseArguments<Options>(args)
-                .WithParsed(opts => options = opts)
-                .WithNotParsed(
-                    errs =>
+            ConfigureLogging();
+            var host = Host.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration(
+                    (hostBuilderContext, configbuilder) =>
                     {
-                        Console.WriteLine("Invalid options were entered.");
-                        Console.WriteLine(errs.ToString());
-                        result = ReturnStatus.Error;
-                    });
-
-            if (result != ReturnStatus.Success)
-            {
-                return (int) result;
-            }
-
-            if (!string.IsNullOrEmpty(options.ConfigurationFilePath))
+                        configbuilder.SetBasePath(hostBuilderContext.HostingEnvironment.ContentRootPath)
+                            .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                            .AddJsonFile($"appsettings.{hostBuilderContext.HostingEnvironment.EnvironmentName}.json", optional: true)
```

Comment:
we really wont need this, however we will be adding another json in the future to address sqlserver vs postgres. i am still working on  a design